docs/benchmarking.md

```diff
@@ -0,0 +1,183 @@
+it will automatically grab the latest model from the ml-compiler-opt Github.
+If it is set to "" or "autogenerate", it will use the autogenerated model.
+* `output_path` - The path to the output file (in JSON format)
+* `perf_counter` - A flag that can be specified multiple times that takes in
+performance counters in the libpfm format. There can only be up to three
+performance counters specified due to underlying limitations in Google
+benchmark.
+* `tensorflow_c_lib_path` - The path to the tensorflow c library if you aren't
+doing an incremental LLVM build.
+* `tests_to_run` - This specifies the LLVM microbenchmarks to run relative to
+the microbenchmarks library in the LLVM test suite build directory. The default
+values for this flag should be pretty safe and produce good results.
+
+You can also get detailed information on each flag by only passing the `--help`
+flag to the script. You can also see the default values here as a lot of the
+flags set in the example above are just to their default value.
+
+## Benchmarking Chromium
+
+You can use the `benchmark_chromium.py` script in order to run chromium
+benchmarks based on test description JSON files.
+
+Example:
+```bash
+cd ~/ml-compiler-opt
+PYTHONPATH=$PYTHONPATH:. python3 ./compiler_opt/tools/benchmark_chromium.py \
+  --advisor=release \
+  --chromium_build_path=./out/Release \
+  --chromium_src_path=~/chromim/src \
+  --compile_llvm \
+  --compile_tests \
+  --depot_tools_path=~/depot_tools \
+  --llvm_build_path=~/llvm-build \
+  --llvm_source_path=~/llvm-project/llvm \
+  --nollvm_use_incremental \
+  --model_path="download" \
+  --num_threads=32 \
+  --output_file=./output-chromium-testing.json \
+  --perf_counters=mem_uops_retired.all_loads \
+  --perf_counters=mem_uops_retired.all_stores \
+  --tensorflow_c_lib_path=~/tensorflow
+```
+
+Several of the flags here are extremely similar to/the same as the flags
+for the llvm test suite, so only the flags unique to the chromium script
+will be highlighted here.
+* `--chromium_build_path` - The path to place the chromium build in. This path
+is relative to the chromium source path.
+* `-chromium_src_path` - The path to the root of the chromium repository (ie
+`./src/` where you ran `fetch --nohooks`)
+* `--depot_tools_path` - The path yo your depot tools checkout.
+* `--num_threads` - enables parallelism for running the tests. Make sure to use
+this with caution as it can add a lot of noise to your benchmarks depending
+upon what specifically you are doing.
+* `--perf_counters` - similar to the llvm test suite perf counters, but instead
+of being in the libpfm format, they're perf counters as listed in `perf list`.
+* `--test_description` - Can be declared multiple times if you have custom test
+descriptions that you want to run, but the default works well, covers a broad
+portion of the codebase, and has been specifically designed to minimize run
+to run variability.
+
+## Generating Chromium Test Descriptions
```

Comment:
make it ### so that it is under Benchmarking Chromium, and also add the explanation about it in your PR description to the md file here

Reply:
Adjusted the heading level and added in all the relevant info from the PR description into the file.